src/routes/index.svelte

```diff
@@ -13,3 +16,6 @@
 
 <HeroHeader />
 <Features />
+<Trusted />
```

Comment:
Let's use a more clear name for these two components (e.g. `TrustedBy`, `CallToAction`).